comparison.md

```diff
@@ -56,7 +56,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 			<tr>
 				<td><strong>24x7 infrastructure and application monitoring</strong></td>
 				<td>✅</td>
-				<td>⚠️ Only on Elite plan</td>
+				<td>✅ New Relic APM Pro free for all sites. Additional monitoring for Elite plan</td>
```

Comment:
we actually mean here that the application is monitored for the client by the Hoster, so I updated slightly that it's a self-serve system: https://github.com/amazeeio/docs/pull/16/commits/81732b9c926652ecb7eb83c5149c6ee352a15caf